packages/shortest/src/ai/prompts/index.ts

```diff
@@ -42,6 +43,14 @@ IMPORTANT GLOBAL RULES:
    - All expectations listed in the test instructions must be fulfilled.
    - If any expectation is not met, the test case must be marked as failed.
 
+8. **Testing Email**:
+   - If you need to test a condition that involves seeing the contents of an email, use the "check_email" tool.
+   - For email validation, you MUST always use 'Click' and 'Mouse' action instead of using keyboard shortcuts.
+   - This tool will grab the latest email from the email address given to you and will render it in a new tab for you to see.
+   - Once you are done with validating the email, navigate back to the original tab.
```

Comment:
It would close the email preview tab automatically?

Reply:
No it does not but it will change the browser tab to the origin.